### ISSUE TITLE
Print Preview Failing

### ISSUE THREAD

**Issue by jj3666** [flathub/org.gnucash.GnuCash#8]
Debian 9, fully updated.

When attempting to print preview a check I receive the following error.
![gnucash_print_preview](https://user-images.githubusercontent.com/421542/36443390-71c11e04-162d-11e8-901d-81a8b1ef6c3b.png)

**Comment by jj3666** [comment]
The above error window no longer shows. Had some more time to run with --debug --extra --logto stdout flags and found the following when clicking print preview for a check.

* 11:07:23  WARN <> Couldn't connect to D-Bus system bus, Could not connect: No such file or directory
* 11:07:24  WARN <Gtk> Error launching preview Failed to execute child process “evince” (No such file or directory)

Evince is installed. Wrong path? or outside sandbox perms for DBUS? Just guessing.
$ evince --version
GNOME Document Viewer 3.22.1

**Comment by TingPing** [comment]
This may chance with the 3.0 update coming soon which moves to Gtk3.

**Comment by TingPing** [comment]
Oh actually looking at your original error you may not have `xdg-desktop-portal-gtk` installed anyway.

**Comment by jj3666** [comment]
Yeah, tried xdg stuff originally and saw it was outdated. When the newer gnome and xdg paks were updated I lost the pop up error and looked deeper. I'll await 3.0, thanks for the response.

**Comment by gjanssens** [comment]
With the latest flatpak updates (to get 3.6) I don't see any error anymore at all. But unfortunately print preview for checks is still not working either.
